api/libproject/devstart.rb

```diff
@@ -659,17 +659,30 @@ def make_bq_denormalized_tables(*args)
   :fn => ->(*args) { make_bq_denormalized_tables(*args) }
 })
 
-def generate_cdr_counts(*args)
+def generate_private_cdr_counts(*args)
   common = Common.new
-  common.run_inline %W{docker-compose run db-generate-cdr-counts} + args
+  common.run_inline %W{docker-compose run db-generate-private-cdr-counts} + args
 end
 
+def generate_public_cdr_counts(*args)
+  common = Common.new
+  common.run_inline %W{docker-compose run db-generate-public-cdr-counts} + args
+end
+
+Common.register_command({
+  :invocation => "generate-private-cdr-counts",
+  :description => "generate-private-cdr-counts --bq-project <PROJECT> --bq-dataset <DATASET> --workbench-project <PROJECT> \
+ --cdr-version=<''|YYYYMMDD> --bucket <BUCKET>
+Generates databases in bigquery with data from a cdr that will be imported to mysql/cloudsql to be used by workbench.",
```

Comment:
Could you add a note here about the inputs being deidentified (and a similar comment on the public command).

Reply:
Done.